app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1186,9 +1187,11 @@ public void updateMarker(boolean isBookmarked, Place place, @Nullable fr.free.nr
     }
 
     /**
-     * Greys out all markers except current location marker
+     * Removes all markers except current location marker, an icon has been used
+     * but it is transparent more than grey(as the name of the icon might suggest)
+     * since grey icon may lead the users to believe that it is disabled or prohibited contribution
      */
-    private void greyOutAllMarkers() {
+    private void removeAllMarkers() {
```

Comment:
As mentioned in [your comment](https://github.com/commons-app/apps-android-commons/pull/3407#issuecomment-585461677) you're just *hiding* the pins rather than removing them. So, wouldn't it be better to name this `hideAllMarkers`?

Since we're there. I also think the `All` part of the name is misleading as it gives the person reading the code the incorrect idea that we would be hiding all the marker while we're only hiding the markers which aren't close to current location. I think `hideOtherLocationMarkers` conveys that in a better way.

Reply:
@sivaraam that's a nice suggestion I'll change this